### PR TITLE
Update for dvl A125

### PR DESCRIPTION
**Pull request by Williangalvani**

### CODE REVIEW

File: dvl-a50/static/service.json
Comment (rjehangir) [inline dvl-a50/static/service.json:1]: "Water Linked" with a space and capital L.

File: README.md
Comment (rjehangir) [inline README.md:1]: "Water Linked" with a space and capital L.

File: dvl-a50/dvl.py
Comment (rjehangir) [inline dvl-a50/dvl.py:1]: "Water Linked" with a space and capital L.

File: dvl-a50/static/service.json
Comment (rjehangir) [reply dvl-a50/static/service.json:1]: "Driver for Water Linked DVLs"

File: dvl-a50/dvl.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Code for integration of Waterlinked DVL A50 with Companion and ArduSub
+Code for integration of Waterlinked DVL A50/A125 with Companion and ArduSub
```
Comment (rjehangir) [inline dvl-a50/dvl.py]: Maybe we should say "BlueOS" instead of Companion. lol

**Comment by Williangalvani** [comment]
@rjehangir I pushed a new commit fixing all "Waterlinked"

**Review by rjehangir — APPROVED** [review]

**Pull request merged**